llm4pddl/envs/assets/pddl/dressed/generate_tasks.py

```diff
@@ -0,0 +1,199 @@
+"""Generates dressed pddl problems."""
+
+from typing import List
+
+import numpy as np
+
+
+def _generate_dressing_problems(min_people: int, max_people: int,
+                                min_casual: int, max_casual: int,
+                                min_formal_dress: int, max_formal_dress: int,
+                                min_formal_suit: int, max_formal_suit: int,
+                                min_extra_clothes: int, max_extra_clothes: int,
+                                num_probs: int,
+                                rng: np.random.Generator) -> List[str]:
+    """Generates problems from a distribution.
+
+    max numbers are exclusive, so if you want exactly two people set min
+    to 2 and max to 3.
+    """
+    problems = []
+    for _ in range(1, num_probs + 1):
```

Comment:
minor: since the counting variable isn't being used here, might as well just do `range(num_probs)` instead